### ISSUE TITLE
CloudFormation 500 error

### ISSUE THREAD

**Issue by taylorfinnell** [localstack/localstack#1768]
Using the latest docker image I get the following during a serverless deploy

```
localstack_1  | 2019-11-17T22:10:38:ERROR:localstack.services.cloudformation.cloudformation_starter: Unable to parse and create resource "ApiUnderscorecrawlerUnderscoredeleteLambdaVersiong1ugB5NU8yFQUYFhFxm7Oi5nsglZkVqvc5D4clmtJi8": 'FunctionName' Traceback (most recent call last):
localstack_1  |   File "/opt/code/localstack/localstack/services/cloudformation/cloudformation_starter.py", line 173, in parse_and_create_resource
localstack_1  |     return _parse_and_create_resource(logical_id, resource_json, resources_map, region_name)
localstack_1  |   File "/opt/code/localstack/localstack/services/cloudformation/cloudformation_starter.py", line 229, in _parse_and_create_resource
localstack_1  |     resource_json_arns_fixed, resources_map, region_name)
localstack_1  |   File "/opt/code/localstack/.venv/lib/python3.7/site-packages/moto/cloudformation/parsing.py", line 306, in parse_and_create_resource
localstack_1  |     resource_name, resource_json, region_name)
localstack_1  |   File "/opt/code/localstack/.venv/lib/python3.7/site-packages/moto/awslambda/models.py", line 506, in create_from_cloudformation_json
localstack_1  |     function_name = properties['FunctionName']
localstack_1  | KeyError: 'FunctionName'```

**Comment by whummer** [comment]
Thanks for reporting @taylorfinnell . Can you please provide a minimal example (serverless config) to reproduce this issue? Thanks

**Comment by taylorfinnell** [comment]
I will try to get a minimal example. In the mean time, I noticed that the function name in the error is 92 characters.

**Comment by whummer** [comment]
@taylorfinnell Has this issue been solved in the meantime - can you please provide a short update here, for reference? Thanks

**Comment by taylorfinnell** [comment]
@whummer I am not exactly sure what fixed it but in the latest release 0.10.6 I no longer have the issue.
